docs/client/session.md

```diff
@@ -0,0 +1,25 @@
+# Semantic conventions for session
+This document defines semantic conventions to apply to client-side applications when tracking sessions.
+
+A session is collection of Logs, Events, and Spans associated with a specific device or user within a specific period of time. 
+A session is represented by a unique identifier that is attached to Logs, Events, and Spans as an attribute.
+
+The primary purpose of sessions is to provide insight into the series of user actions or events that lead a critical or 
+interesting occurrence in the application's use. Sessions also provide a means to quantify overall application usage.
+
+
+## Attributes
+
+This attribute can be applied to all signals generated by an agent. It may be omitted from metrics, due to their aggregated nature. 
+
+| Attribute   | Type    | Description | Examples                             | Requirement level | 
+|-------------|---------|-------------|--------------------------------------|-------------------|
+| session.id  |  String | A GUID      | 00112233-4455-6677-8899-aabbccddeeff | required          |
```

Comment:
Can you add this in a yaml at https://github.com/open-telemetry/semantic-conventions/tree/main/model? Since it's a new top level namespace I guess it could be in a file named session.yaml

Reply:
Do we intend `session` to be a namespace, so that other attributes related to session can be added in the future? If not, and this is a single attribute, then I think it should be `session_id` instead.

Reply:
@martinkuba  what would be the namespace of `session_id`? There's an `enduser` namespace, does it make sense to add under that?

Reply:
Does it need to be a GUID or just something that has a high cardinality and is expected to be unique across some domain or namespace?

I think an optional domain/namespace column makes sense here as there could be different types of sessions that we'd want to track different (e.g. background, foreground, etc.). Having an optional value allows us to do that without adding extra burden for those that don't care.

Reply:
I wonder if we should introduce a `client` namespace? I worry `enduser` implies it's for authenticated users?